packages/devtools_app/lib/src/ui/service_extension_widgets.dart

```diff
@@ -143,7 +145,9 @@ class _ServiceExtensionButtonGroupState
               : 0.0,
         ),
         child: ImageIconLabel(
-          description.icon,
+          extensionState.isSelected
+              ? description.enabledIcon
+              : description.disabledIcon,
```

Comment:
what if either are null? If we make these fields required (one or both - see earlier comment about making disabledIcon default to the value of enabledIcon when null), we can avoid this consideration

Reply:
Yep, I will make `enabledIcon` required and default `disabledIcon` to it if not specified